cosmos-sdk-proto/src/prost/tendermint.p2p.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
@jkilpatr when you say:

> I notice the 'tendermint' files are not currently being used in the repo should we bother to check in the rust files in that case?

Were you talking about this file, or something else?

Clearly this file is bogus and should probably be excluded.

Reply:
it seems that all the files starting with tendermint.* are totally unused and not compiled.